### ISSUE TITLE
Network

### ISSUE THREAD

**Issue by llorllale** [zold-io/java-api#5]
Let's implement `Network`:

https://github.com/zold-io/java-api/blob/f323e05f461a45ebe4a8b791abbc567be7165f8e/src/main/java/io/zold/api/Network.java#L34-L48

**Comment by llorllale** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/zold-io/java-api/issues/5#issuecomment-400077095))

@llorllale Job [#5](https://github.com/zold-io/java-api/issues/5) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by paulodamaso** [comment]
@llorllale Just a doubt here, whitepaper says in page 13 about `Fetch` action, it is related to `Network.pull(id)` method, right? It wouldn't be better to keep naming consistency and use `fetch` for the method name?

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@llorllale Please take a look at this: `pull` is supposed to get the merged wallets from the network?

**Comment by llorllale** [comment]
@paulodamaso sorry for the delay - I somehow forgot about this question.

I have doubts myself - please see zold-io/zold#387

**Comment by 0crat** [comment]
> @0crat wait for #31 merge ([here](https://github.com/zold-io/java-api/issues/5#issuecomment-404385742))

@paulodamaso The impediment for [#5](https://github.com/zold-io/java-api/issues/5) was registered successfully by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0pdd** [comment]
@llorllale 4 puzzles [#38](https://github.com/zold-io/java-api/issues/38), [#39](https://github.com/zold-io/java-api/issues/39), [#40](https://github.com/zold-io/java-api/issues/40), [#41](https://github.com/zold-io/java-api/issues/41) are still not solved.

**Comment by llorllale** [comment]
@paulodamaso thanks

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@d96641de-c711-445a-a096-857ab360f1f5, thread: AsyncFlush-4
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 1 error(s) in XML document: -1:-1: Duplicate unique value [gh:zold-io/java-api#5] declared for identity constraint of element "verbosity".
```

[0.26.3](https://github.com/zerocracy/farm): CID: [2001511415](https://www.0crat.com/CAZUREFND/2001511415), Type: "Close job", Author: "llorllale"

**Comment by 0crat** [comment]
Job `gh:zold-io/java-api#5` is not assigned, can't get start time

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#5](https://github.com/zold-io/java-api/issues/5) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +46 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@llorllale 4 puzzles [#38](https://github.com/zold-io/java-api/issues/38), [#39](https://github.com/zold-io/java-api/issues/39), [#40](https://github.com/zold-io/java-api/issues/40), [#56](https://github.com/zold-io/java-api/issues/56) are still not solved; solved: [#41](https://github.com/zold-io/java-api/issues/41).

**Comment by 0pdd** [comment]
@llorllale 4 puzzles [#38](https://github.com/zold-io/java-api/issues/38), [#39](https://github.com/zold-io/java-api/issues/39), [#56](https://github.com/zold-io/java-api/issues/56), [#61](https://github.com/zold-io/java-api/issues/61) are still not solved; solved: [#40](https://github.com/zold-io/java-api/issues/40), [#41](https://github.com/zold-io/java-api/issues/41).

**Comment by 0pdd** [comment]
@llorllale 5 puzzles [#38](https://github.com/zold-io/java-api/issues/38), [#39](https://github.com/zold-io/java-api/issues/39), [#61](https://github.com/zold-io/java-api/issues/61), [#79](https://github.com/zold-io/java-api/issues/79), [#80](https://github.com/zold-io/java-api/issues/80) are still not solved; solved: [#40](https://github.com/zold-io/java-api/issues/40), [#41](https://github.com/zold-io/java-api/issues/41), [#56](https://github.com/zold-io/java-api/issues/56).

**Comment by 0pdd** [comment]
@llorllale 8 puzzles [#38](https://github.com/zold-io/java-api/issues/38), [#39](https://github.com/zold-io/java-api/issues/39), [#79](https://github.com/zold-io/java-api/issues/79), [#80](https://github.com/zold-io/java-api/issues/80), [#81](https://github.com/zold-io/java-api/issues/81), [#82](https://github.com/zold-io/java-api/issues/82), [#83](https://github.com/zold-io/java-api/issues/83), [#84](https://github.com/zold-io/java-api/issues/84) are still not solved; solved: [#40](https://github.com/zold-io/java-api/issues/40), [#41](https://github.com/zold-io/java-api/issues/41), [#56](https://github.com/zold-io/java-api/issues/56), [#61](https://github.com/zold-io/java-api/issues/61).

**Comment by 0pdd** [comment]
@llorllale 8 puzzles [#116](https://github.com/zold-io/java-api/issues/116), [#117](https://github.com/zold-io/java-api/issues/117), [#38](https://github.com/zold-io/java-api/issues/38), [#39](https://github.com/zold-io/java-api/issues/39), [#79](https://github.com/zold-io/java-api/issues/79), [#80](https://github.com/zold-io/java-api/issues/80), [#82](https://github.com/zold-io/java-api/issues/82), [#84](https://github.com/zold-io/java-api/issues/84) are still not solved; solved: [#40](https://github.com/zold-io/java-api/issues/40), [#41](https://github.com/zold-io/java-api/issues/41), [#56](https://github.com/zold-io/java-api/issues/56), [#61](https://github.com/zold-io/java-api/issues/61), [#81](https://github.com/zold-io/java-api/issues/81), [#83](https://github.com/zold-io/java-api/issues/83).
